mitosheet/mitosheet/steps_manager.py

```diff
@@ -185,9 +187,13 @@ def __init__(self, args: Collection[Union[pd.DataFrame, str]]):
                 preprocess_step_performers.preprocess_step_type()
             ] = execution_data
 
+        initial_args: Dict[int, pd.DataFrame] = OrderedDict()
```

Comment:
Add a comment explaining what is happening here - also do we want to move this up?